scripts/core/editor3/reducers/tests/suggestion_style.spec.jsx

```diff
@@ -141,7 +141,7 @@ describe('editor3.reducers.suggestion.CREATE_CHANGE_STYLE_SUGGESTION', () => {
         }
     });
 
-    it('CREATE_CHANGE_STYLE_SUGGESTION - set BOLD partially overlapping suggestions', () => {
+    it('should overwrite overlapped section when set 2 BOLD overlapped suggestions', () => {
```

Comment:
Clarify the description, I can't understand what does it do from the description alone

Reply:
fix is in the next PR